django/chat/static/chat/js/scripts.js

```diff
@@ -137,11 +137,11 @@ document.addEventListener("DOMContentLoaded", function () {
 // On prompt form submit...
 document.addEventListener("htmx:afterSwap", function (event) {
   if (event.target.id != "messages-container") return;
-  if (event.detail.pathInfo.requestPath.includes('upload'))
-    return;
   if (document.querySelector("#no-messages-placeholder") !== null) {
     document.querySelector("#no-messages-placeholder").remove();
   }
+  if (event.detail.pathInfo.requestPath.includes('upload'))
+    return;
```

Comment:
Needed to move this down here to ensure that the placeholder is removed when user uploads a file to a new chat.